components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -2247,3 +2263,23 @@ type TeamMembershipInvite struct {
 	InvalidationTime string         `json:"invalidationTime,omitempty"`
 	InvitedEmail     string         `json:"invitedEmail,omitempty"`
 }
+
+type Ordering string
+
+const (
+	Ordering_Ascending  Ordering = "asc"
+	Ordering_Descending Ordering = "desc"
+)
+
+type AdminGetTeamsOptions struct {
+	Offset     int      `json:"offset,omitempty"`
+	Limit      int      `json:"limit,omitempty"`
+	OrderBy    int      `json:"orderBy,omitempty"`
+	Order      Ordering `json:"orderDir,omitempty"`
+	SearchTerm string   `json:"searchTerm,omitempty"`
+}
+
+type AdminGetTeamsResult struct {
```

Comment:
I wanted to make this a generic type, but sadly our mock codegen does not yet support generics - it's worked on in https://github.com/golang/mock/pull/669